addons/hr_recruitment/views/hr_applicant_views.xml

```diff
@@ -5,11 +5,12 @@
         <field name="name">Applicants</field>
         <field name="model">hr.applicant</field>
         <field name="arch" type="xml">
-            <tree string="Applicants" multi_edit="1" sample="1">
+            <tree string="Applicants" multi_edit="1" sample="1" decoration-danger="application_status == 'refused'">
                 <field name="message_needaction" invisible="1"/>
                 <field name="last_stage_id" invisible="1"/>
                 <field name="date_last_stage_update" invisible="1"/>
                 <field name="partner_name" readonly="1" optional="show"/>
+                <field name="create_date" readonly="1" widget="date" optional="show" string="Applied on"/>
```

Comment:
Wouldn't it be more consistent to change the field string instead?

Reply:
done